### DISCUSSION TITLE
Sponsoring section with no links should not be visible.

### DISCUSSION THREAD

**Discussion by sueannioanis** [community/community#15430] [Other Feature Feedback, Questions, & Ideas]
**Bug (with screen capture)**: Empty sponsoring section after sponsoring for a month.

<img width="351" alt="bug" src="https://user-images.githubusercontent.com/3586765/164865230-e8ad1b23-65c8-4059-a378-15cc612ae2bb.png">

**Expected Behaviour**: No section when there is nothing populated.

**Comment by sueannioanis** [comment]
I was able to resolve the issue by sponsoring another repository I like.

**Comment by jlord** [comment]
Thanks for reporting this, I'll create a ticket so the team can look into it 👍
